packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -427,15 +430,34 @@ export const makeTreeEditGenerator = (
 		}
 	}
 
+	const constraintGenerator = createWeightedGenerator<NodeConstraint, FuzzTestState>([
+		[
+			(state): NodeConstraint => ({
+				type: "nodeConstraint",
+				content: maybeDownPathFromNode(
+					// Selecting the parent node here, since the field is possibly empty.
+					selectTreeField(viewFromState(state), state.random, weights.fieldSelection).content
```

Comment:
This seems fine for now, although I do wonder if we should refactor things to have a separate `selectTreeNode` function.